integration_tests/web/test_admin_barriers.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+import logging
+import os
+import unittest
+
+from integration_tests.env_variable_names import (
+    SLACK_SDK_TEST_GRID_WORKSPACE_ADMIN_USER_TOKEN,
+    SLACK_SDK_TEST_GRID_ORG_ADMIN_USER_TOKEN,
+    SLACK_SDK_TEST_GRID_IDP_USERGROUP_ID,
+    SLACK_SDK_TEST_GRID_TEAM_ID,
+)
+from integration_tests.helpers import async_test
+from slack import WebClient
+
+
+class TestWebClient(unittest.TestCase):
+    """Runs integration tests with real Slack API"""
+
+    def setUp(self):
+        self.logger = logging.getLogger(__name__)
+        self.org_admin_token = os.environ[SLACK_SDK_TEST_GRID_ORG_ADMIN_USER_TOKEN]
+        self.sync_client: WebClient = WebClient(
```

Comment:
I know this is still testing the legacy client but it's okay so far. I'm going to migrate most of the tests to the new package soon - I will create a ticket for it.

Reply:
here is the issue: https://github.com/slackapi/python-slack-sdk/issues/895